### PR TITLE
Neutral interaction

### PR DESCRIPTION
**Pull request by asztalosors**
The neutral calculation has been added. Tests were performed and added.
Current neutral DB is not considered final as cross-sections might evolve.
Regression testing will be postponed upon the addition of the final cross-section DB.

### CODE REVIEW

File: utility/input.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            self.property_labels.append('temperature')
+            self.unit_labels.append('eV')
+            self.profile_data.append(temperature)
+
+    def get_beamlet_input(self):
+        self._build_components()
+        self._build_profiles()
+        return self.param, self.components, self.profiles
+
+    def from_text(self, source, charges, atomic_numbers, mass_numbers, molecular_numbers):
```
Comment (gergopokol) [inline utility/input.py:121]: This should be removed and new issue created to do this.

File: visualization/profiles.py [hunk 2]
```diff
@@ -136,8 +142,9 @@ def benchmark(self, benchmark_param_path='../data/beamlet/IMAS_beamlet_test_prof
         matplotlib.pyplot.show()
 
     def __setup_density_axis(self, axis):
-        axis.plot(self.profiles['beamlet grid'], self.profiles['electron']
-                  ['density']['m-3'], label='Density', color='b')
+        for comp in self.components.T.keys():
+            axis.plot(self.profiles['beamlet grid'], self.profiles[str(comp)]
+                      ['density']['m-3'], label=str(comp), color='b')
```
Comment (gergopokol) [inline visualization/profiles.py:147]: color should not be specified

**Review by gergopokol — CHANGES_REQUESTED** [review]
Minor changes requested.

File: utility/input.py (hunk 1 above)
Comment (asztalosors) [reply utility/input.py]: Done

File: visualization/profiles.py (hunk 2 above)
Comment (asztalosors) [reply visualization/profiles.py]: Done

**Review by gergopokol — APPROVED** [review]
Reviewed, approved.

**Pull request merged**